community/hazelcast-jet-enterprise/templates/management-center-deployment.yaml

```diff
@@ -0,0 +1,102 @@
+{{- if .Values.managementcenter.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "hazelcast-jet-management-center.fullname" . }}
+  labels:
+    app: {{ template "hazelcast-jet.name" . }}
+    chart: {{ template "hazelcast-jet.chart" . }}
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "hazelcast-jet.name" . }}
+      chart: {{ template "hazelcast-jet.chart" . }}
```

Comment:
Using versions in a deployment's label selector is discouraged since they will be immutable after getting created and helm upgrade will break if there is a version change.  See https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#label-selector-updates for more details.

Reply:
Version excluded from the template